graph/src/components/store/entity_cache.rs

```diff
@@ -374,7 +374,9 @@ impl EntityCache {
         let mut entity = entity;
         let old_vid = entity.set_vid(vid).expect("the vid should be set");
         // Make sure that there was no VID previously set for this entity.
-        assert!(old_vid.is_none());
+        if old_vid.is_some() {
+            bail!("VID was already present when set in EntityCache");
```

Comment:
This is very very minor but in these error messages it's very helpful if it includes the entity that is involved (type and id) and what the old vid was.